app/src/main/java/org/coepi/android/ui/container/TabsContainerFragment.kt

```diff
@@ -5,13 +5,7 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.Fragment
-import androidx.navigation.Navigation
-import androidx.navigation.ui.setupWithNavController
-import com.google.android.material.bottomnavigation.BottomNavigationView
-import org.coepi.android.R.id.bottom_navigation
-import org.coepi.android.R.id.navHostFragment
 import org.coepi.android.databinding.FragmentTabsContainerBinding.inflate
-import org.coepi.android.ui.onboarding.OnboardingViewModel
 import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class TabsContainerFragment : Fragment() {
```

Comment:
Rename it in `ContainerFragment`. Probably the class and navigation root should be removed, but let's keep it for now.